src/detections/chart/center-message.jsx

```diff
@@ -0,0 +1,11 @@
+import { useStyles } from 'detections/chart/style'
+import { Grid } from '@material-ui/core'
+import React from 'react'
+import PropTypes from 'prop-types'
+
+export const CenterMessage = ({ children }) => <Grid container justify={'center'} direction={'column'}
```

Comment:
Let's extract these props to a separate useProps hook.

Reply:
done